badger/cmd/write_bench.go

```diff
@@ -415,3 +419,38 @@ func dropAll(c *y.Closer, db *badger.DB) {
 		}
 	}
 }
+
+func dropPrefix(c *y.Closer, db *badger.DB) {
+	defer c.Done()
+
+	dropPeriod, err := time.ParseDuration(dropPrefixPeriod)
+	y.Check(err)
+	if dropPeriod == 0 {
+		return
+		log.Println("Exit from drop prefix")
+	}
+
+	t := time.NewTicker(dropPeriod)
+	defer t.Stop()
+	for {
+		select {
+		case <-c.HasBeenClosed():
+			return
+		case <-t.C:
+			fmt.Println("[DropPrefix] Started")
+			err := db.DropAll()
+			for err == badger.ErrBlockedWrites {
```

Comment:
Why would drop prefix see blocked writes? This could only happen if `dropPeriod < time taken to drop the prefix`. I don't think we should do this. Do a simple error check here and stop the program if `err != nil`

Reply:
Should we just return, or panic?